core/lib/libtomcrypt/src/ciphers/aes/aes_modes_armv8a_ce_a64.S

```diff
@@ -466,6 +467,7 @@ ENTRY(ce_aes_ctr_encrypt)
 	adds            x6, x6, #1              /* increment BE ctr */
 	rev             x7, x6
 	ins             v4.d[1], x7
+	cbnz            x9, .Lctrcarrydone      /* ignore overflow 64-bit ctr */
```

Comment:
How does this deal with different integer values for `ctrlen`?  I mean `ctrlen == 0, 1, 2, 3 ... 16`.
If only a limited number of cases are supported (such as 64 and 128 bits as it seems), it should be stated in the comment near the function prototype.

Reply:
Ok will add comment in function prototype.